community/social.md

```diff
@@ -1,3 +1,25 @@
-# pyOpenSci Social Media
+# pyOpenSci Social Media Platforms
+The 2024 North Star for pyOpenSci’s social media accounts is to raise general awareness of pyOpenSci in the broader open source community through both scheduled and ad hoc content that reflects our commitments to diversity and inclusion within the open source scientific Python community.
```

Comment:
Perhaps also add - to build a contributor / volunteer community that supports our mission of empowering scientists to create better software that supports more open and reproducible science